common/src/main/java/com/redhat/lightblue/rest/RestConfiguration.java

```diff
@@ -74,7 +74,7 @@ public static LightblueFactory getFactory() {
         return factory;
     }
 
-    public static LightblueFactory getFactory(
+    public synchronized static LightblueFactory getFactory(
```

Comment:
Inefficient. Instead:
1) Declare factory as volatile
2) remove synchronized
3) getFactory() {
       LightblueFactory f=factory;
       if(f==null) {
            synchronized(this) {
                    if(factory==null)  initialize factory
                    f=factory;
            }
   return f;
}